src/katgpucbf/utils.py

```diff
@@ -184,6 +186,76 @@ def cancel(self) -> None:
             sensor.detach(self)
 
 
+def _esterror_status(value: float) -> aiokatcp.Sensor.Status:
```

Comment:
Part of me wants to request a change to the variable/docstring-esque comment to remind us that `value` is in seconds.

*But*, I know the `units` are specified in the sensor itself. It's not like someone else is going to randomly use this (and `_maxerror_status`) for a difference `status_func`, right?

Reply:
It's also just generally good practice to use base units internally and convert to things like milliseconds only for display, so in general I don't worry too much about documenting the units of things when they're in base units (which has lead to the code-base containing a mix of `adc_sample_rate` and `adc_sample_rate_hz`). On the other hand, `value` isn't a name that adds a lot of value (ha!), so I'll rename it to `seconds` to make it self-documenting.